pyproject.toml

```diff
@@ -32,9 +30,9 @@ dependencies = [
     "jsonschema",
     "particle",
     "tqdm >=4.24.0", # autonotebook
-    'python-constraint2; python_version >="3.8.0"',
```

Comment:
See **[PEP&nbsp;508](https://peps.python.org/pep-0508/#specification)**. This means that the package is only installed if your Python version is at least 3.8. For this requirement, this `python_version` is not required anymore, since you already require the installer to have at least Python 3.9.